lib/streamlit/file_util.py

```diff
@@ -122,13 +124,34 @@ def get_static_dir():
     return os.path.normpath(os.path.join(dirname, "static"))
 
 
+def get_main_script_path() -> Path:
+    import sys
+
+    # Script path should always be added to path, see _fix_sys_path method
+    if sys.argv[0].endswith(".py"):
+        return Path(sys.argv[0])
+    else:
+        for arg_name in sys.argv:
+            if arg_name.endswith(".py"):
+                return Path(arg_name)
+
+    return Path(sys.argv[0])
+
+
 def get_app_static_dir(main_script_path: str) -> str:
     """Get the folder where app static files live"""
     main_script_path = Path(main_script_path)
     static_dir = main_script_path.parent / APP_STATIC_FOLDER_NAME
     return os.path.abspath(static_dir)
 
 
+def get_app_locale_dir() -> str:
+    """Get the folder where app locale files live"""
+    main_script_path = get_main_script_path()
+    locale_dir = main_script_path.parent / APP_LOCALE_FOLDER_NAME
+    return os.path.abspath(locale_dir)
```

Comment:
```suggestion
    locale_dir = (main_script_path.parent / APP_LOCALE_FOLDER_NAME).absolute()
    return str(locale_dir)
```
The Python API has two path API, so we should choose one and use it consistently, at least for the scope of one function.

Reply:
It's updated and now I return str from every function inside `file_util.py` 👍